batchflow/sampler.py

```diff
@@ -45,13 +46,12 @@ def _get_method_by_alias(alias, module, tf_distributions=None):
 
     return fullname
 
+
```

Comment:
Why `ALIASES` and `_get_method_by_alias` are not part of the base `Sampler` class?

Reply:
As well as the `MAX_ITERS` constant